simple_generator.py

```diff
@@ -38,41 +36,52 @@
 MyMIDI.addTrackName(track, time, "Sample Track")
 MyMIDI.addTempo(track, time, 120)
 random.seed()
-binfile = open(file, 'wb')
 
 
+# returns list of lists, which are melodies
 def make_music_rand():
-    seq = []
-    seq_ac = []
+    seq_seq = []
+    seq = [0]
+    seq_ac = [1]
+    seq_ac2 = [1]
     i = 0
-    while i < 99:
+    while i < 21:
         a = random.randint(1, 7)
         seq.append(a)
         i = i + 1
 
     i = 0
-    while i < 99:
+    while i < 21:
         a = random.randint(1, 7)
         seq_ac.append(a)
         i = i + 3
-    write_sequence_ac(seq_ac)
-    return seq
 
+    i = 0
+    while i < 21:
+        a = random.randint(1, 7)
+        seq_ac2.append(a)
+        i = i + 3
+
+    seq_seq.append(seq)
+    seq_seq.append(seq_ac)
+    seq_seq.append(seq_ac2)
+    return seq_seq
 
-# now we expect list here
-def write_sequence(args):
+
+# we expect list here
+def write_sequence(melody):
     print("writing sequence")
     order = 0
-    for arg in args:
-        write_note(switch(arg), order, 100)
+    for note_pos in range(1, len(melody)):
```

Comment:
that's bad. 
If we iterate over the collection it's better use `for item in col` (`for note in melody` here) than trying to emulate C-like for loop.

Reply:
nice fix there